tools/xcodeproj_shims/install.sh

```diff
@@ -12,6 +12,19 @@ case "${PRODUCT_TYPE}" in
     com.apple.product-type.framework)
         input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${TARGET_NAME}/${FULL_PRODUCT_NAME}")
         ;;
+    com.apple.product-type.framework.static)
+	    # For static library, as the output is not under bazel-bin, we have to get it based on build event
+        # Example of such entry inside build event:
+        # important_output {
+        #   name: ".../SomeFramework.framework/SomeFramework"
+        #   uri: "file:///private/var/tmp/.../.../SomeFramework.framework/SomeFramework"
+        #   path_prefix:... 
+        #   ...
+        #  } 
+        # We only care about the entry ending with TARGET_NAME" so that we can get the path to its directory
+        QUERY=`grep -A 2 important_output "$BAZEL_BUILD_EVENT_TEXT_FILENAME" | grep -w uri | grep ${TARGET_NAME}\" | sed "s/uri: \"file:\/\///"`
```

Comment:
grep ${TARGET_NAME}\" is what I have concern with, because right now in order to get the right path that is single and deterministic, I am only looking for the one path ending with the binary file (which is usually just the target name, but it can be different if user sets it to be a diff. thing?)

Reply:
In the future, we may want to parse the the build_event_json_file.